workspace/utilities/com/site-header-mobile-toggler.xsl

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+	<xsl:template name="site-header-mobile-toggler">
+		<xsl:param name="ext-attr" />
+
+		<!-- STYLES /////////////////////////////////////////////////////////////// -->
+
+		<xsl:variable name="attr-site-header-mobile-toggler">
+			<add class="block" />
+			<add class="square-broad" />
+			<add class="cursor-pointer" />
+
+			<add class="js-change-state-click" />
+			<add data-change-state-click="mobile-opened" />
+			<add data-change-state-action="toggle" />
+
+			<add data-mobile-opened-state-follower=".js-site-header-mobile-toggler-icon-part, .js-site-header" />
+			<add data-mobile-opened-state-follower-common-ancestor="#site" />
+
+			<xsl:copy-of select="$ext-attr"/>
+			<add dev-component="site-header-mobile-toggler" />
+		</xsl:variable>
+
+		<xsl:variable name="attr-inner-ctn">
+			<add class="flexbox flex-row align-items-center" />
+		</xsl:variable>
+
+		<xsl:variable name="attr-icon">
+			<add class="block" />
+			<add class="relative" />
+			<add class="width-broad height-thinner" />
+		</xsl:variable>
+
+		<xsl:variable name="attr-icon-part">
+			<add class="block" />
+			<add class="absolute left" />
+			<add class="bg-color-current-color" />
+			<add class="width-full height-2px" />
```

Comment:
I don't think `height-2px` is generated. Should we use styles in this case or add a note in comments on top of the file?

Reply:
Same thing it seems for `bg-color-current-color`.

Reply:
Current color is auto generated in class-colors.less by default

Reply:
But true that height-2px is not generated by default

Reply:
> Current color is auto generated in class-colors.less by default

Oh that's where it waaaaas !

Reply:
@PascalPiche @alexnantel88  So what do you prefer for the height? 
1. Use style
2. Comment on top of file to warn of custom class
3. Let it as is

Reply:
4. It can be a style attribute but with a xslt param for the value.

Reply:
Done !